src/zeek-setup.cc

```diff
@@ -660,6 +660,16 @@ SetupResult setup(int argc, char** argv, Options* zopts)
 	init_net_var();
 	run_bif_initializers();
 
+	// Assign the script_args for command line processing in Zeek scripts.
+	if ( ! options.script_args.empty() )
+		{
+		auto script_args_val = zeek::id::find_val("zeek_script_args")->AsVectorVal();
+		for ( const string& script_arg: options.script_args )
+			{
+			script_args_val->Assign(script_args_val->Size(), zeek::make_intrusive<zeek::StringVal>(script_arg));
```

Comment:
```suggestion
			script_args_val->Assign(script_args_val->Size(), make_intrusive<StringVal>(script_arg));
```

Doesn't really matter, but previous agreements were to avoid unnecessary namespace qualification to improve readability.

Reply:
Thanks for all of this feedback. I'll be able to do things better in the future this way at least.